docs/en/04_Changelogs/3.7.0.md

```diff
@@ -0,0 +1,26 @@
+# 3.7.0
+
+### Versioned cache segmentation
```

Comment:
@unclecheese Can you please document this in a persistent place in the docs as well? Also apply this to the 4.x PRs

Reply:
@unclecheese Still outstanding